src/components/PlantCard.js

```diff
@@ -1,18 +1,42 @@
-import React from "react";
+import React, { useState } from "react"
+import PriceChange from "./PriceChange"
+
+function PlantCard({
+  plant: { id, name, image, price },
+  onPlantDelete,
+  onUpdatePrice,
+}) {
+  const [isInStock, setIsInStock] = useState(true)
+
+  function handleInStock() {
+    setIsInStock(!isInStock)
+  }
+
+  function handleDeleteClick() {
+    fetch(`http://localhost:6001/plants/${id}`, {
+      method: "DELETE",
+    })
+      .then((response) => response.json())
+      .catch((error) => console.error("Fetch error:", error))
+    onPlantDelete(id)
+  }
 
-function PlantCard() {
   return (
     <li className="card">
-      <img src={"https://via.placeholder.com/400"} alt={"plant name"} />
-      <h4>{"plant name"}</h4>
-      <p>Price: {"plant price"}</p>
-      {true ? (
-        <button className="primary">In Stock</button>
+      <img src={image} alt={name} />
+      <h4>{name}</h4>
+      <p>Price: {price}</p>
+      <PriceChange id={id} currentPrice={price} onUpdatePrice={onUpdatePrice} />
+      {isInStock ? (
+        <button className="primary" onClick={handleInStock}>
+          In Stock
+        </button>
       ) : (
-        <button>Out of Stock</button>
+        <button onClick={handleInStock}>Out of Stock</button>
       )}
```

Comment:
Instead of wrapping everything in a conditional, you could also just have the class name in a conditional and the text in a conditional. Either way is probably fine though.